obvspython/vis.py

```diff
@@ -33,3 +34,41 @@ def create_heatmap(model_1_layers, model_2_layers, values) -> go.Figure:
     )
 
     return fig
+
+
+def create_annotated_heatmap(values: List[float], cell_annotations: List[str], x_ticks: List[str],
```

Comment:
lovely. did you try making the other standard logit lens visses?

Reply:
Not yet, but would love to in the coming sprints.